data-storage-stats/src/lib.rs

```diff
@@ -0,0 +1,277 @@
+// Copyright 2023 Salesforce, Inc. All rights reserved.
+
+mod constants;
+
+use constants::*;
+use pdk::data_storage::{DataStorage, DataStorageBuilder, StoreMode};
+use pdk::hl::*;
+use pdk::logger;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use std::rc::Rc;
+use std::time::{SystemTime, UNIX_EPOCH};
+
+/// Configuration parameters for the data storage stats policy.
+#[derive(Deserialize, Clone)]
+struct Config {
```

Comment:
this was manually modified. you need to enrich the gcl yaml, this will be auto updated when running make build asset files